www/privoxy/Portfile

```diff
@@ -53,10 +53,12 @@ if {[option gpg_verify.use_gpg_verification]} {
     }
 }
 
-worksrcdir          ${name}-${version}-stable
+set perl5_major_min 5.28
+if {${perl5.major} eq "" || ${perl5.major} < ${perl5_major_min}} {
+    perl5.major     ${perl5_major_min}
+}
 
-depends_build-append \
-                    port:p${perl5.major}-sgmlspm
```

Comment:
are you sure you don't need this dependency anymore?

Reply:
Yes, that’s to build the man page, which doesn’t work because of some missing `openjade` files.